PWGHF/TableProducer/HFCandidateCreatorChic.cxx

```diff
@@ -233,12 +233,12 @@ struct HFCandidateCreatorChicMC {
 
         int indexMother = RecoDecay::getMother(particlesMC, particlesMC.iteratorAt(indexRec), pdg::Code::kChic1);
         int indexMotherGamma = RecoDecay::getMother(particlesMC, particlesMC.iteratorAt(candidate.index1().mcparticleId()), pdg::Code::kChic1);
-        if (indexMother > -1 && indexMotherGamma == indexMother && candidate.index1().mcparticle_as<aod::McParticles_000>().pdgCode() == kGamma) {
+        if (indexMother > -1 && indexMotherGamma == indexMother && candidate.index1().mcparticle_as<aod::McParticles>().pdgCode() == kGamma) {
           auto particleMother = particlesMC.iteratorAt(indexMother);
           hEphotonMatched->Fill(candidate.index1().e());
           hMassEMatched->Fill(sqrt(candidate.index1().px() * candidate.index1().px() + candidate.index1().py() * candidate.index1().py() + candidate.index1().pz() * candidate.index1().pz()));
-          int indexDaughterFirst = particleMother.daughter0Id(); // index of the first direct daughter
-          int indexDaughterLast = particleMother.daughter1Id();  // index of the last direct daughter
+          int indexDaughterFirst = particleMother.daughtersIds().front(); // index of the first direct daughter
+          int indexDaughterLast = particleMother.daughtersIds().back();   // index of the last direct daughter
           if ((indexDaughterFirst > -1 && indexDaughterLast > -1)) {
             std::vector<int> arrAllDaughtersIndex;
             RecoDecay::getDaughters(particlesMC, particleMother, &arrAllDaughtersIndex, array{(int)(kGamma), (int)(pdg::Code::kJpsi)}, 1);
```

Comment:
@vkucera Shall we also change the syntax of this function? Converting internally now in a pair of numbers where `daughterIds()` is already a pair of numbers seems not needed...

Reply:
Replace by just `has_daughters()`